src/lib/Scenes/Artwork/Components/ArtworkTombstone.tsx

```diff
@@ -124,11 +124,15 @@ export class ArtworkTombstone extends React.Component<ArtworkTombstoneProps, Art
           </Serif>
         )}
         {artwork.attribution_class && (
-          <Text style={{ textDecorationLine: "underline" }} onPress={() => this.handleTap("/artwork-classifications")}>
-            <Serif color="black60" size="3t">
+          <Serif color="black60" size="3t" mt={1}>
+            <Text
+              style={{ textDecorationLine: "underline" }}
+              onPress={() => this.handleTap("/artwork-classifications")}
```

Comment:
Maybe we should add this `onPress` on a `TouchableWithoutFeedback` element instead of binding it to a `Text` element.

Reply:
I used text so I could add the underline. Are you suggesting I wrap the text element in a TouchableWithoutFeedback?

Reply:
Yea exactly! Just for the `onPress`